perllib/FixMyStreet/App/Controller/Admin.pm

```diff
@@ -1884,6 +1898,35 @@ sub user_hide_everywhere : Private {
     $c->stash->{status_message} = _('That user’s reports and updates have been hidden.');
 }
 
+sub send_login_email : Private {
+    my ( $self, $c, $email, $name, $user_id ) = @_;
+
+    my $token_data;
+    $token_data = {
+        email => $email,
+    };
+
+    if ( $user_id ) {
+        $token_data->{old_user_id} = $user_id;
+    }
+
+    $token_data->{name} = $name if $name;
```

Comment:
If I'm being really pernickety (3yo was annoying earlier), odd to have this and above be two different ways of doing the same thing :)